dojo/tools/semgrep/parser.py

```diff
@@ -71,3 +71,15 @@ def convert_severity(self, val):
             return "Info"
         else:
             raise ValueError(f"Unknown value for severity: {val}")
+
+    def get_description(self, item):
+        description = ''
+
+        message = item["extra"]["message"]
+        description += '**Result message:** {}\n'.format(message)
+
+        snippet = item["extra"]["lines"]
```

Comment:
```suggestion
        snippet = item["extra"].get("lines")
```

Using `get` will be safer, if `lines` shouldn't be in the report.

Reply:
Fixed, thanks for the suggestion.